src/main/features/claimant-response/views/counter-offer-accepted.njk

```diff
@@ -20,13 +20,15 @@
               courtOrderFrequency: courtOrderPaymentPlan.paymentSchedule | renderPaymentSchedule
```

Comment:
Can't this be null/undefined here? you have a check on line 23 but not here.

Reply:
Potentially yes, well spotted